client/my-sites/plans/ecommerce-trial/upgrade-confirmation/index.tsx

```diff
@@ -43,16 +42,12 @@ const TrialUpgradeConfirmation = () => {
 	} );
 
 	const currentPlanName = isFetchingSitePlan ? '' : selectedSite?.plan?.product_name_short ?? '';
-	const isEnrepreneurTrialFlow = config.isEnabled( 'entrepreneur-my-home' );
```

Comment:
We still need a check but a one based on the billing plan rather than on the flag.